apps/svelte.dev/content/blog/2025-03-01-whats-new-in-svelte-march-2025.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "What's new in Svelte: March 2025"
+description: 'Congrats to the SvelteHack winners! Plus improved SSR in Svelte and SvelteKit'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+[SvelteHack 2024](https://hack.sveltesociety.dev/2024) has ended and the winners were announced on February 9th. Congrats to all the [winners](https://hack.sveltesociety.dev/2024/winners) who you'll also find listed in the "Community Showcase" below.
+
+But before we dive into that, let's get to some updates!
+
+## What's new in Svelte and SvelteKit
+
```

Comment:
you could mention web sockets are available for testing: https://github.com/sveltejs/kit/issues/1491#issuecomment-2645962690